static/app/utils/replays/frame.tsx

```diff
@@ -125,6 +130,8 @@ export function getTitle(frame: ReplayFrame): ReactNode {
   if ('category' in frame) {
     const [type, action] = frame.category.split('.');
     switch (frame.category) {
+      case 'replay.init':
+        return 'Replay Init';
```

Comment:
Init sounds a bit too technical, maybe Start?

```suggestion
        return 'Replay Start';
```

Reply:
got it in https://github.com/getsentry/sentry/pull/53189